src/scss/components/selector-strip/_hxSelector.scss

```diff
@@ -4,6 +4,7 @@
 .hxSelector {
   border-radius: 2px;
   color: $cyan-900;
```

Comment:
Please use updated  `v2.0` colors to replace `$cyan-XXX`.

Reply:
Changes have been done, please review.